tests/testthat/test-object_usage_linter.R

```diff
@@ -363,3 +363,19 @@ test_that("interprets glue expressions", {
     }
   "), "local_var", object_usage_linter(interpret_glue = FALSE))
 })
+
+# reported as #1088
+test_that("definition in parent environment is detected", {
```

Comment:
This doesn't really describe what is happening at the moment.

Reply:
true... wrote the test long before the issue was diagnosed. updated.